SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -272,39 +268,22 @@ void SSpatialOutputLog::StartPollTimer(const FString& LogFilePath)
 
 void SSpatialOutputLog::FormatAndPrintRawErrorLine(const FString& LogLine)
 {
-	const FRegexPattern ErrorPattern = FRegexPattern(TEXT("level=(.*) msg=(.*) code=(.*) code_string=(.*) error=(.*) stack=(.*)"));
-	FRegexMatcher ErrorMatcher(ErrorPattern, LogLine);
+	FString LogCategory = "";
 
-	if (!ErrorMatcher.FindNext())
+	if (ErrorLogFlagInfo.Key)
 	{
-		UE_LOG(LogSpatialOutputLog, Error, TEXT("Failed to parse log line: %s"), *LogLine);
-		return;
+		LogCategory = ErrorLogFlagInfo.Value;
```

Comment:
You can't have an empty category. The else here should probably just be "Runtime"